release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -302,6 +302,16 @@ The latency test, a part of the CNF-test container, provides a way to measure if
 
 For information about running a latency test, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-performing-end-to-end-tests-running-the-latency_tests[Running the latency tests].
 
+[id="ocp-4-7-scale-new-vrf-cni-plugin"]
+==== New VRF CNI plugin allows secondary networks to be assigned to VRFs
+
+A new VRF CNI plugin is now available which allows you to assign additional networks to a VRF. When you create a secondary network using a `rawConfig` configuration for the CNO custom resource and configure a VRF for it, the interface created for the pod is associated with the VRF. You can also use the VRF CNI plug-in to assign an SR-IOV network to a VRF.
```

Comment:
plug-in is now available, which

Reply:
hmm. I have a small quibble with this. I don't think the first clause takes a comma. 

All I can find in the IBM style guide is the following: "Use which, preceded by a comma, to introduce a nonrestrictive clause. A nonrestrictive clause provides extra information that is not essential to the meaning of the sentence. If the nonrestrictive clause is removed, the basic meaning of the sentence does not change."

This is not a nonrestrictive clause...

Reply:
Ah, okay. My hypercorrection 🙂. Sorry! As long as plugin is updated to plug-in we should be good to go!